src/utils.ts

```diff
@@ -748,3 +749,32 @@ export function setCrypto(c: Object) {
 export function getCrypto(): Object {
     return crypto;
 }
+
+export async function retryNetworkOperation(maxAttempts, callback) {
```

Comment:
Maybe more natural to put this in `http-api` as well, since it's meant to be used for network things...?

Reply:
I wasn't sure if we can do general request retrying without side-effects. I'd rather be conservative and do it just for account data to begin with.

Reply:
Oh, that's fine and I agree it's risky for general requests and good to be conservative. I was just reacting to the placement in `utils.ts` when it seems fairly specific to network activity, so was just suggesting moving this function into the file `http-api.js` instead of here.

Reply:
Ah, sure :+1: